############################################################
## ErrorNavigation.msg
## Mensaje que contiene 2 campos:
##      - float32[]  errorPos: error de posicion xyz
##      - float32    errorRot: error de rotacion sobre eje Z
############################################################
float32[] errorPos
float32 errorRot